src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt

```diff
@@ -60,6 +73,15 @@ class IntegrationsPatch : BytecodePatch(
                 )
             ).toMutable()
         )
+
+        val serviceIndex = serviceMethod.implementation!!.instructions.lastIndex
+        serviceMethod.addInstructions(
+            serviceIndex, """
+               invoke-static {}, Lcom/google/android/apps/youtube/app/YouTubeTikTokRoot_Application;->getAppContext()Landroid/content/Context;
+               move-result-object p0
```

Comment:
Is `p0` safe for use here? Just making sure :)

Reply:
its not a "fresh" register but since its being overwritten it should be fine
i should've got the register dynamically tho

Reply:
You don't need to get it dynamically per see. I was asking if you're not overriding a register that's being used after without setting a value first, indirectly causing a use-after-free.

Reply:
keep in mind that PR is still a draft tho

Reply:
I'm aware.